recipes/minimap2/build.sh

```diff
@@ -22,3 +22,9 @@ cp minimap2 misc/paftools.js ${PREFIX}/bin
 cp sdust ${PREFIX}/bin
 cp libminimap2.a ${PREFIX}/lib
 cp *.h ${PREFIX}/include
+
+if [ ! -d $PREFIX/share/man/man1 ] ; then
```

Comment:
There is no need of the check. 
mkdir -p would not fail if the folder already exists